### PR TITLE
Fix skybox visibility in certain situations 

### PR DESCRIPTION
**Pull request by heymanMC**
- Don't show skybox underwater (vanilla behavior)
- Don't show skybox when player has darkness effect

### CODE REVIEW

File: src/main/java/io/github/amerebagatelle/fabricskyboxes/skyboxes/AbstractSkybox.java [hunk 1]
```diff
@@ -218,11 +218,14 @@ protected boolean checkEffect() {
...
 
         CameraSubmersionType cameraSubmersionType = camera.getSubmersionType();
-        if (cameraSubmersionType == CameraSubmersionType.POWDER_SNOW || cameraSubmersionType == CameraSubmersionType.LAVA)
+        if (cameraSubmersionType == CameraSubmersionType.POWDER_SNOW || cameraSubmersionType == CameraSubmersionType.LAVA || camera.getSubmersionType() == CameraSubmersionType.WATER)
             return false;
 
         if (camera.getFocusedEntity() instanceof LivingEntity livingEntity && livingEntity.hasStatusEffect(StatusEffects.BLINDNESS))
             return false;
+        
+        if (camera.getFocusedEntity() instanceof LivingEntity livingEntity && livingEntity.hasStatusEffect(StatusEffects.DARKNESS))
```
Comment (AMereBagatelle) [inline src/main/java/io/github/amerebagatelle/fabricskyboxes/skyboxes/AbstractSkybox.java]: This change was superseded by #44.

**Review by AMereBagatelle — CHANGES_REQUESTED** [review]
Makes sense, however #44 already fixed the darkness issue.

**Comment by RaptaG** [comment]
> Don't show skybox underwater (vanilla behavior)

Can this be toggleable?

**Comment by heymanMC** [comment]
Will redo PR with only the underwater change.

**Pull request closed without merging**